packages/server/graphql/mutations/helpers/notifications/SlackNotifier.ts

```diff
@@ -250,6 +251,24 @@ export const SlackSingleChannelNotifier: NotificationIntegrationHelper<SlackNoti
   async integrationUpdated() {
     // Slack sends a system message on its own
     return 'success'
+  },
+
+  async standupResponseSubmitted(meeting, team, user, response) {
+    const options = {
+      searchParams: {
+        utm_source: 'slack share',
+        utm_medium: 'product',
+        utm_campaign: 'sharing',
+        responseId: response.id
+      }
+    }
+    const responseUrl = makeAppURL(appOrigin, `meet/${meeting.id}/responses`, options)
```

Comment:
I came here as the metrics police👮‍♂️ and gladly see you have already done what I would have asked for! ❤️